### PR TITLE
UI Improved

### PR DESCRIPTION
**Pull request by skpandey885**
#377 

UI Improved and Responsiveness Improved!

### CODE REVIEW

**Comment by skpandey885** [comment]
Its shows that my branch has some conflicts can you help me how to solve this conflict issue!
@QAZIMAAZARSHAD

**Comment by QAZIMAAZARSHAD** [comment]
> Its shows that my branch has some conflicts can you help me how to solve this conflict issue!
> @QAZIMAAZARSHAD

Fetch Upstream

**Comment by QAZIMAAZARSHAD** [comment]
Also share the deployment link so I can review your changes.

**Comment by skpandey885** [comment]
Bhaiya the latest Changes Are Not reflecting In The Deployment link! That is the reason you were telling me that day to fetch upstream! i dont know why this is happening its just showing the chanhes it did 4 days ago and after that whatever changes im making its not reflecting there

**Comment by QAZIMAAZARSHAD** [comment]
> Bhaiya the latest Changes Are Not reflecting In The Deployment link! That is the reason you were telling me that day to fetch upstream! i dont know why this is happening its just showing the chanhes it did 4 days ago and after that whatever changes im making its not reflecting there

It's because you are still a few commits behind.

Try to fetch upstream then see.
Screenshot (109)

**Comment by skpandey885** [comment]
I did it multiple times

**Comment by QAZIMAAZARSHAD** [comment]
Never forget to share the deployment link.

**Comment by skpandey885** [comment]
> Never forget to share the deployment link.
Bhaiya I think I m having some issues! I deleted the environment link and I'm cloning it again and pushing it and will generate a fresh pr!

**Comment by QAZIMAAZARSHAD** [comment]
> > Never forget to share the deployment link.
> 
> Bhaiya I think I m having some issues! I deleted the environment link and I'm cloning it again and pushing it and will generate a fresh pr!

Fine.

I thought you are done.

**Comment by skpandey885** [comment]
> > > Never forget to share the deployment link.
> > 
> > 
> > Bhaiya I think I m having some issues! I deleted the environment link and I'm cloning it again and pushing it and will generate a fresh pr!
> 
> Fine.
> 
> I thought you are done.

I don't know why i am facing issues

**Comment by QAZIMAAZARSHAD** [comment]
> > > > Never forget to share the deployment link.
> > > 
> > > 
> > > Bhaiya I think I m having some issues! I deleted the environment link and I'm cloning it again and pushing it and will generate a fresh pr!
> > 
> > 
> > Fine.
> > I thought you are done.
> 
> I don't know why i am facing issues

Start over again. This is the best way.

If we want to create something good we must prepare to throw our first work.

**Comment by QAZIMAAZARSHAD** [comment]
If you need any help @kanak22 will help you.

**Comment by skpandey885** [comment]
bhaiya! I am seeing these errors from yesterday...check your repo of help.html file in vs code!  

Screenshot 2021-07-24 at 12 24 07 PM

**Comment by skpandey885** [comment]
Bhaiya you merge this pr! bcz this pr has all the changes I made! now again I have to do all steps !

File: help.html [hunk 1]
```diff
@@ -220,20 +243,23 @@ <h2><Strong>Details</Strong></h2>
                     </script>
                 </div>
                 <div class="help-right">
-                    <h2><strong>Reach Us</strong></h2>
-                    <br>
+
+                    <h2 style="text-align: center; text-decoration: underline; font-family: 'Baloo Tammudu 2', cursive;">REACH US</h2>
```
Comment (nlok5923) [inline help.html:247]: @skpandey885 you had used inline styling most of the place please convert it to class based styling.

**Review by nlok5923 — CHANGES_REQUESTED** [review]
thanks PTAL !!

**Pull request closed without merging**